src/zkscript/groth16/model/groth16.py

```diff
@@ -207,7 +207,7 @@ def groth16_verifier_unlock(
             M = int(log2(r)) if max_multipliers is None else int(log2(max_multipliers[i]))
 
             if pub[i] == 0:
-                out += Script.parse_string("OP_1") + Script.parse_string(" ".join(["OP_0", "OP_0"] * M))
+                out += Script.parse_string("OP_1") + Script.parse_string(" ".join(["OP_0"] * M))
```

Comment:
It's because the function `unrolled_multiplication` from `EllipticCurveUnrolledFq` has changed, and therefore this part of the code of Groth16 (which is the unlocking script for `unrolled_multiplication`) must be changed accordingly.